numba/targets/arraymath.py

```diff
@@ -1372,3 +1373,44 @@ def histogram_impl(a, bins, range=None):
             return hist, bins
 
     return histogram_impl
+
+
+# Create np.finfo, np.iinfo and np.MachAr
+# machar
+_mach_ar_supported = ('ibeta', 'it', 'machep', 'eps', 'negep', 'epsneg',
+                      'iexp', 'minexp', 'xmin', 'maxexp', 'xmax', 'irnd',
+                      'ngrd', 'epsilon', 'tiny', 'huge', 'precision',
+                      'resolution',)
+MachAr = namedtuple('MachAr', _mach_ar_supported)
+
+# Do not support MachAr field
+# finfo
+_finfo_supported = ('bits', 'eps', 'epsneg', 'iexp', 'machep', 'max',
+                    'maxexp', 'negep', 'nexp', 'nmant', 'precision',
```

Comment:
Yes! Well spotted. Not quite sure how that happened as I pulled them from a `dir()`!? Fixed in 43fd4c4